README.md

```diff
@@ -587,6 +588,26 @@ migration "state" "test" {
 }
 ```
 
+#### state xmv
+
+The `xmv` command works like the `mv` command but allows usage of
+wildcards `*` in the source definition. The source expressions will be 
+matched against resources defined in the terraform state. The matched value
+can be used in the destination definition via a dollar sign and their ordinal number: 
+`$1`, `$2`, ... When there is ambiguity the ordinal number can be put in curly braces (e.g. `${1}`).
+
+For example if `foo` and `bar` in the `mv` command example above are the only 2 security group resources
+defined at the top level then you can rename them using:
+
+```hcl
+migration "state" "test" {
+  dir = "dir1"
+  actions = [
+    "xmv aws_security_group.* aws_security_group.${1}2",
```

Comment:
I tested this in a sandbox environment and found that the HCL parser treats `$` as a special symbol, so we need to be escaped with `$$`.

```
"xmv aws_security_group.* aws_security_group.$${1}2",
```